pallets/liquid-staking/src/lib.rs

```diff
@@ -309,9 +317,13 @@ pub mod pallet {
             amount: Balance,
         ) -> DispatchResultWithPostInfo {
             T::WithdrawOrigin::ensure_origin(origin)?;
-
+            let left_amount = amount
+                .checked_sub(Self::reserve_factor().mul_floor(amount))
```

Comment:
where does this fee goes?

Reply:
It still leaves on the relaychain :)

Reply:
The fee should record on chain, and if we can put them in staking will be better.